get-latest-tgf.sh

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env bash
+get_local_tgf_version () {
+        TGF_LOCAL_VERSION=$(tgf --current-version | awk -F\  '{print $2}');
+}
+
+get_latest_tgf_version () {
+    if [ -z "$GITHUB_TOKEN" ]; then
+        echo 'GITHUB_TOKEN is not set.';
+    fi
+    TGF_LATEST_VERSION=$(curl --silent https://api.github.com/repos/coveo/tgf/releases/latest?access_token=${GITHUB_TOKEN} | grep tag_name | awk -F\" '{print $4}');
+}
+
+done_script () {
+    echo "Done.";
+    tgf --current-version;
+    exit 0;
+}
+
+install_latest_tgf () {
+        VERSION=$(echo $TGF_LATEST_VERSION | cut -d'v' -f 2);
+        
+        if [[ "$OSTYPE" == "linux"* ]]; then
```

Comment:
I tried it on Windows Subsystem for Linux and this condition does not work. However, you can run `uname -s` which will return "Linux" or "Darwin". So, it would look like `if [[ $(uname -s) == Linux ]]`

Reply:
Oh nice 
Thanks !